examples/avmnist_multimodal/configs/README.md

```diff
@@ -0,0 +1,3 @@
+# Configurations
+
+Configurations for experiments using [YAML](https://en.wikipedia.org/wiki/YAML)
```

Comment:
I agree this is good to have. Then please add to all examples not just this.
BTW, missing period in the end.

Reply:
Added this to all examples.